skyreader/result.py

```diff
@@ -946,10 +947,52 @@ def bounding_box(ra, dec, theta, phi):
             contour_colors=['k', 'r', 'g', 'b'][:3]
 
         sample_points = np.array([np.pi/2 - grid_dec, grid_ra]).T
+        
+        def circular_contour(ra, dec, sigma, nside):
+                """For plotting circular contours on skymaps ra, dec, sigma all
+                expected in radians."""
+                dec = np.pi/2. - dec
+                sigma = np.rad2deg(sigma)
+                delta, step, bins = 0, 0, 0
+                delta= sigma/180.0*np.pi
+                step = 1./np.sin(delta)/10.
+                bins = int(360./step)
+                Theta = np.zeros(bins+1, dtype=np.double)
+                Phi = np.zeros(bins+1, dtype=np.double)
+                # define the contour
+                for j in range(0,bins) :
+                    phi = j*step/180.*np.pi
+                    vx = np.cos(phi)*np.sin(ra)*np.sin(delta) + np.cos(ra)*(np.cos(delta)*np.sin(dec) + np.cos(dec)*np.sin(delta)*np.sin(phi))
+                    vy = np.cos(delta)*np.sin(dec)*np.sin(ra) + np.sin(delta)*(-np.cos(ra)*np.cos(phi) + np.cos(dec)*np.sin(ra)*np.sin(phi))
+                    vz = np.cos(dec)*np.cos(delta) - np.sin(dec)*np.sin(delta)*np.sin(phi)
+                    idx = healpy.vec2pix(nside, vx, vy, vz)
+                    DEC, RA = healpy.pix2ang(nside, idx)
+                    Theta[j] = DEC
+                    Phi[j] = RA
+                Theta[bins] = Theta[0]
+                Phi[bins] = Phi[0]
+                return Theta, Phi
+        
         # Call meander module to find contours
-        contours_by_level = meander.spherical_contours(sample_points,
-            grid_value, contour_levels
-            )
+        if not is_rude:
+            contours_by_level = meander.spherical_contours(sample_points,
+                grid_value, contour_levels
+                )
+        if is_rude:
+            sigma50 = np.deg2rad(0.2)
+            sigma90 = np.deg2rad(0.7)
```

Comment:
Hardcoded values should be avoided.

Since these are somewhat conventional for a specific use case ("realtime splinempe") I would prefer providing `skyreader` with a more generic functionality of plotting arbitrarily-sized contours and let the user code (could be your own script or SkyMist) dedide the size.

Reply:
Added as parameters to create_plot_zoomed()